index.html

```diff
@@ -8164,23 +8164,36 @@ <h3 id="Nature"><span class="wip">Nature - Use with Caution</span></h3>
                <td>Beast Shape
                </td>
                <td>You gain the ability to shapeshift into a small animal for 1AP (See Beast
-                  Shape table in the Appendix). This is limited to 3 uses. You also gain the abilities associated
+                  Shape table in the Appendix).  You also gain the abilities associated
                   with that beast. When you shapeshift into the beast, your modifiers and damage
                   changes and you can carry across HP up to that of the max HP of the beast. When
-                  you are reduced to 0HP as the beast, you revert back to your original form minus
+                  you are reduced to 0HP as the beast, you instantly revert back to your original form minus
                   the HP you transferred to the beast - any additional damage is also carried
                   over. You still have/use your pool of resistance points as normal while in your
                   Beast Shape, and you can still cast Nature abilities. You still use your own INT
                   and SPI attributes while in the Beast Shape.
-                  <p>
-                     You can voluntarily leave the Beast Shape at any time on your turn for 0AP, and
-                     any remaining HP gets returned to your normal health pool.
+                  <br><br>
+                  You can voluntarily leave the Beast Shape at any time on your turn for 0AP, and
+                  any remaining HP gets returned to your normal health pool.
                </td>
                <td>Self
                </td>
-               <td>1
-                  <p>
-                     0
+               <td>
+                  1
+                  <br><br>
+                  0
+               </td>
+            </tr>
+            <tr>
+               <td>Core
+               </td>
+               <td>Poison Ivy Whip
+               </td>
+               <td>Do a regular SPI based ranged attack dealing Medium+SPI Poison damage to an enemy within 100ft.
```

Comment:
Cool, I appreciate this. Should it be 50ft range cause Nature is 50ft based rather than 100ft?